src/rnodeAPI.js

```diff
@@ -111,13 +115,16 @@ function RNode(
   assert.ok(host, 'endPoint.host missing');
   assert.ok(port, 'endPoint.port missing');
 
-  const proto = grpc.loadPackageDefinition(packageDefinition);
-  const casper = proto.coop.rchain.casper.protocol;
-
-  const client /*: DeployService */ = new casper.DeployService(
+  const deployProto = grpc.loadPackageDefinition(deployServiceDefinition);
+  const client /*: DeployService */ = new deployProto.coop.rchain.casper.protocol.DeployService(
     `${host}:${port}`, grpc.credentials.createInsecure(), // ISSUE: let caller do secure?
   );
 
+  const proposeProto = grpc.loadPackageDefinition(proposeServiceDefinition);
+  const proposeClient /*: ProposeService */ = new proposeProto.coop.rchain.casper.protocol.ProposeService(
```

Comment:
@nzpr strange thing, in some cases, when `ProposeService` is undefined, `DeployService` exist in the same namespace. I couldn't fine where is the problem.

Tests are passing with something like this.
```js
const ProposeServiceFixed =
  proposeProto.coop.rchain.casper.protocol.ProposeService || 
  proposeProto.coop.rchain.casper.protocol.DeployService;
```

Reply:
@dckc @tgrospic Digging into static codegen, this convo https://github.com/grpc/grpc-node/issues/528 makes me think that static code generated with protobufjs library that rnode-api was using does not allow to actually call gRPC methods. 

> While protobufjs CAN generate code for service definition, it can NOT be used for gRPC, because it'd be lacking a lot of features.

So I'm trying to use protoc compiler from https://github.com/grpc/grpc-node.
Or maybe its a good idea to use runtime proto loader.

Reply:
@nzpr I think you right, protobufjs is just a wrapper around protoc and needs protocol implementation for communication. I tried static generated js with grpc-node but I couldn't find a way to create correct rpc implementation.

There is an example which uses `grpc-tools` but I couldn't make it work to generate grpc (service) part.
https://github.com/grpc/grpc/tree/master/examples/node/static_codegen

But I have some good news also. :tada: With `grpc-web`, Envoy proxy and protoc (with web plugin) I managed to successfully connect to RNode from the browser.
Now I see your point with `Either` type which holds a value of type byte array. Which is like untyped because it must be deserialized to correct type. And this (generic) type of Either is written in comments in proto files e.g. [DeployService](https://github.com/rchain/rchain/blob/6fc87d5fb38be666c86490f3495bde7e631504f0/models/src/main/protobuf/DeployService.proto#L32).

Reply:
Thats is cool!. 🎉 
I switched to static codegen here https://github.com/nzpr/rchain-api/tree/static, and successfully deployed/proposed on live RNode. Try `npm run integrationTest` to check.  But I'm struggling with RHOCore part now and some other problems emerged.

Reply:
@tgrospic Regarding generating gRPC please see https://github.com/nzpr/rchain-api/blob/static/protobuf/Makefile
And client constructions pretty straightforward then https://github.com/nzpr/rchain-api/blob/static/src/rnodeAPI.js#L115

Reply:
Thanks. Now I see that grpc-tools is nodejs version of protoc which accepts the same arguments and generates the same code. Works with web plugin also.

Reply:
@nzpr 
> @dckc @tgrospic Digging into static codegen, this convo [grpc/grpc-node#528](https://github.com/grpc/grpc-node/issues/528) makes me think that static code generated with protobufjs library that rnode-api was using does not allow to actually call gRPC methods.
> 
> > While protobufjs CAN generate code for service definition, it can NOT be used for gRPC, because it'd be lacking a lot of features.

Odd. protobufjs and grpc get along pretty well, in my experience. protobufjs only does serialization, not the RPC part. But I haven't seen any problems with that arrangement.

> So I'm trying to use protoc compiler from https://github.com/grpc/grpc-node.

I tried that. I don't recall why, but I found that it conflicted with my design ideas.